kingpin/actors/rightscale/test/integration_deployment.py

```diff
@@ -0,0 +1,51 @@
+from nose.plugins.attrib import attr
+
+from tornado import testing
+
+from kingpin.actors.rightscale import deployment
+
+
+__author__ = 'Mikhail Simin <mikhail@nextdoor.com>'
+
+
+class IntegrationDeployment(testing.AsyncTestCase):
+
+    integration = True
+
+    def setUp(self, *args, **kwargs):
+        super(IntegrationDeployment, self).setUp(*args, **kwargs)
+        self.deployment_name = 'Kingpin Deployment Integration Test'
+
+    @attr('integration', 'dry')
+    @testing.gen_test()
+    def integration_01a_create_deployment_dry(self):
+        actor = deployment.Create('Integ. Test',
```

Comment:
Make the deployment name more dynamic -- so its unique per test. For example, look at the way we name the ServerArrays in those integration tests.